recipe/meta.yaml

```diff
@@ -16,26 +16,41 @@ build:
 
 requirements:
   host:
-    - python >=3.6
+    - python >=3.8,<3.12
     - pip
   run:
-    - python >=3.6
-    - tiktoken
-    - dataclasses-json
-    - langchain >=0.0.303
-    - sqlalchemy >=2.0.15
-    - numpy
-    - tenacity >=8.2.0,<9.0.0
-    - openai >=0.26.4
-    - pandas
-    - urllib3 <2
-    - fsspec >=2023.5.0
-    - typing_inspect >=0.8.0
-    - typing-extensions >=4.5.0
-    - beautifulsoup4
-    - nest-asyncio
-    - nltk
-    - tree_sitter_languages
+    - python >=3.8<=3.12
+    - SQLAlchemy ={extras = ["asyncio"], version = ">=1.4.49"}
+    - beautifulsoup4 ="^4.12.2"
+    - dataclasses-json ="*"
+    - deprecated =">=1.2.9.3"
+    - fsspec =">=2023.5.0"
+    - httpx ="*"
+    - langchain ={optional = true, version = ">=0.0.303"}
```

Comment:
this syntax is not supported in conda recipes
you could add it under `run_constrained` or create a separate output similar to https://github.com/conda-forge/matplotlib-feedstock/blob/1fd5929d7f5e3b508cb58ba4e1e9b6cbf5ac3c0c/recipe/meta.yaml#L86-L90